test/powershell/engine/Basic/DefaultConfirmImpact.Tests.ps1

```diff
@@ -0,0 +1,256 @@
+Describe 'Default Cmdlet ConfirmImpact Ratings' -Tags 'CI' {
+    BeforeAll {
+        $moduleList = @(
+            "Microsoft.PowerShell.Utility"
+            "Microsoft.PowerShell.Management"
+            "Microsoft.PowerShell.Security"
+            "Microsoft.PowerShell.Host"
+            "Microsoft.PowerShell.Diagnostics"
+            "PSWorkflow"
```

Comment:
Why is PSWorkflow here?

Reply:
The test should works on Windows PowerShell and contains not ported Windows PowerShell cmdlets - so we can see that is difference PowerShell Core and Windows PowerShell..